### PR TITLE
refactor(web): use global component instead of scoped one

### PR DESCRIPTION
**Pull request by FredericEspiau**
MANAGER-4273

Signed-off-by: Frederic Espiau <frederic.espiau@corp.ovh.com>
Co-authored-by: Jérémy De-Cesare <jeremy.de-cesare@corp.ovh.com>
Co-authored-by: Axel Peter <axel.peter@corp.ovh.com>

### CODE REVIEW

**Comment by marie-j** [comment]
@FredericEspiau A warning though we'll need to sync with other PRs that uses current component (most of existing MIAW features I think)

**Comment by FredericEspiau** [comment]
@marie-j absolutely, because it's used a lot lately (hence why the testing should be done once merging on `develop` 😉 )

File: packages/manager/apps/web/client/app/app.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import ovhManagerCore from '@ovh-ux/manager-core';
 import ngAtInternet from '@ovh-ux/ng-at-internet';
 import ngAtInternetUiRouterPlugin from '@ovh-ux/ng-at-internet-ui-router-plugin';
 import ngOvhApiWrappers from '@ovh-ux/ng-ovh-api-wrappers';
+import ngOvhContracts from '@ovh-ux/ng-ovh-contracts';
```
Comment (FredericEspiau) [inline packages/manager/apps/web/client/app/app.js]: try

**Comment by antleblanc** [comment]
Hi @FredericEspiau!

We recently merged #1336 which apply Prettier rules so could you please rebase
your branch on `develop`?

To easily resolve conflicts you can **ignore** `HEAD` changes and only keep yours.

After that you can run:

```sh
$ yarn run lint
```

This command will lint the entire codebase.

or:

Amend your commit to trigger the `pre-commit` hook (prettier will be launched
under the hood).

Thanks!

**Comment by FredericEspiau** [comment]
done @antleblanc

**Comment by JeremyDec** [comment]
@FredericEspiau conflicts !

**Comment by ovh-cds** [comment]
CDS Report installation#8548.0 ■   
* Stage 1
  * Install ■

**Pull request merged**